histoqc/LightDarkModule.py

```diff
@@ -58,18 +58,31 @@ def getIntensityThresholdPercent(s, params):
     lower_thresh = float(params.get("lower_threshold", -float("inf")))
     upper_thresh = float(params.get("upper_threshold", float("inf")))
 
-    lower_var = float(params.get("lower_variance", -float("inf")))
-    upper_var = float(params.get("upper_variance", float("inf")))
+    # Prepare parameter names due to issues #213 and #219
+    # set lower standard deviation
+    if params.get('lower_var'):
```

Comment:
yup, looks good and is really the only way to ensure backward compatibility due to my bug. 

can we add a warning to the log that lower_var will be deprecated in future versions in favor of lower_std?

we can then justify cleaning up the code in the future